scripts/run

```diff
@@ -51,17 +54,24 @@ RUNTYPE=${1}
 
 run_tests() {
   cd "${ROOT_DIR}"
+  eval "$("${SCRIPTS_DIR}"/render-configs.sh -e)"
```

Comment:
This is needed for both unit and connected tests as well, since these depend on `GOOGLE_APPLICATION_CREDENTIALS` to provide the bean `tdrServiceAccountEmail`.